README.md

```diff
@@ -1,6 +1,9 @@
 # NLP Primitives
 
 <p align="center">
+    <a href="https://codecov.io/gh/alteryx/nlp_primitives">
+        <img src="https://codecov.io/gh/alteryx/nlp_primitives/branch/main/graph/badge.svg"/>
```

Comment:
The badge isn't displaying any coverage stats. Maybe that is to be expected at this stage?

![image](https://user-images.githubusercontent.com/4307001/180870488-a4aa59d9-c65b-49bf-8cf9-6fe653cc8cfd.png)

Reply:
That is expected, it only displays for `main`